cmd/bootstrap/transit/cmd/upload_transit_keys.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/bootstrap/gcs"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// pushTransitKeysCmd represents a command to upload public keys to the transit server
+var pushTransitKeysCmd = &cobra.Command{
+	Use:   "push-transit-keys",
```

Comment:
```suggestion
	Use:   "push-transit-key",
```

We are only pushing one key.